app/web/features/profile/locales/en.json

```
@@ -87,7 +95,16 @@
     "pronouns": "Pronouns",
     "hometown": "Grew up in",
     "occupation": "Occupation",
-    "education": "Education"
+    "education": "Education",
+    "basic_information": "Basic Information",
+    "basic_information_subtitle": "Start with the essentials - your photo and basic details",
+    "preferences": "Preferences",
+    "preferences_subtitle": "Let others know about your hosting and meetup preferences",
+    "personal_information": "Personal Information",
+    "personal_information_subtitle": "Share details about yourself and your background",
+    "about_me_subtitle": "Tell others about yourself - this is what makes your profile unique",
```

I'm definitely doing the link wrong here, but you get the idea. We probably should force this link to open in a separate tab so they don't lose their spot on the edit profile page or trigger the warning?)

```suggestion
    "about_me_subtitle": "Tell others about yourself. This is where trust begins. Help others get a sense of who you really are. See [tips for creating a great profile](https://help.couchers.org/hc/couchersorg-help-center/articles/1725919197-how-do-i-create-a-great-profile)",
```